README.md

```diff
@@ -1,2 +1,58 @@
-# protfolio-webpage
-This is our fifth real project
+# Protfolio- Webpage
+
+## Project Description 📝
+
+> This project is our final project in this project we work from all project that we worked 
+
+
+
+## Demo 📸
+
+
+![Demo](https://github.com/royajamal/protfolio-webpage/assets/168343975/66e64e0d-b3d9-44d4-a297-b78ab66b6ac9)
+
+## Technologies Used 🛠
+
+the technologies that i used to develop my project. is HTML and CSS.
+
+Examples:
+
+- HTML
+- CSS
+
+## Installation 💻
+
+(bash)
+('''# git clone https://github.com/royajamal/protfolio-webpage.git''')
+(vscode-github)
```

Comment:
- Remove these from here:
`(bash)` - Remove this
`('''# git clone https://github.com/royajamal/protfolio-webpage.git''')` - Remove the parenthesis from here `(  )`
`(vscode-github)` - Remove this

the result would be like this without parenthesis:

```
(```
# git clone https://github.com/royajamal/protfolio-webpage.git
''')
```